tests/regression/test_regression.py

```diff
@@ -44,6 +44,7 @@ def _test_regression(
         fxt_tags: dict,
         fxt_num_repeat: int,
         tmpdir: pytest.TempdirFactory,
+        head_name: str = "head",
```

Comment:
How about making it more explicit? We need to change the Classification's test cases, however, it is more clear for me because only classification tasks might use the `head` for the `head_name`.

```suggestion
        head_name: str
```

Reply:
[17d1425](https://github.com/openvinotoolkit/training_extensions/pull/2752/commits/17d14252664efcf3bc432f5c1975eeb8664f1e50) Done!